src/pool/pool_disjoint.c

```diff
@@ -1073,8 +1073,8 @@ umfDisjointPoolParamsCreate(umf_disjoint_pool_params_handle_t *hParams) {
     }
 
     *params = (umf_disjoint_pool_params_t){
-        .slab_min_size = 0,
-        .max_poolable_size = 0,
+        .slab_min_size = 64 * 1024,           // 64K
+        .max_poolable_size = 2 * 1024 * 1024, // 2MB
         .capacity = 0,
```

Comment:
not sure if non zero default would be better (1?) @bratpiorka

Reply:
please take default settings from here ("host" config): https://github.com/oneapi-src/unified-runtime/blob/568a96aabc6edabe8514ae163aecc64cd5a41878/source/common/umf_pools/disjoint_pool_config_parser.cpp#L50